types.bal

```diff
@@ -17,11 +17,7 @@ type Beneficiary record {
 };
 
 type Donor record {
-    string id;
```

Comment:
Let's get donor-Id as well, as this is useful for future interactions.

Reply:
Don't need it yet.